interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/nodecompilation/FragmentParameterValidator.scala

```diff
@@ -35,27 +38,77 @@ class FragmentParameterValidator(
       fragmentParameter.name
     )
 
+    val validationExpressionResult =
+      compileExpressionValidator(fragmentParameter, validationContext(fragmentParameter.name))
+
     List(
       unsupportedFixedValuesValidationResult,
       fixedValuesListValidationResult,
-      fixedExpressionsValidationResult
-    ).sequence.map(_ => ())
+      fixedExpressionsValidationResult,
+      validationExpressionResult
+    ).sequence.map(_.flatten)
+  }
+
+  private def compileExpressionValidator(
+      fragmentParameter: FragmentParameter,
+      typ: TypingResult
+  )(implicit nodeId: NodeId) = fragmentParameter.valueCompileTimeValidation match {
+    case Some(expr) =>
+      if (expr.validationExpression.expression.isBlank) {
```

Comment:
Instead of checking `isBlank`, check if `typedExpression.expression` is an instance of `NullExpression`. Thanks to that it will work better with other expression languages and we will keep this `isBlank` logic in one place